app/components/elements/button/index.js

```diff
@@ -1,15 +1,3 @@
-import {Link} from '/components/elements/link'
+import BaseButton from './base'
 
-const Button = ({to, className = '', children}) =>
-  <Link to={to} class={'btn ' + className}>
-    {children}
-  </Link>
-
-const FormButtom = ({className = '', type, children, ...props}) =>
-  <button type={type} class={'btn ' + className} {...props}>{children}</button>
-
-export default function ButtonHoC ({to, type, children, ...props}) {
-  return !type
-    ? <Button to={to} {...props}>{children}</Button>
-    : <FormButtom to={to} type={type} {...props}>{children}</FormButtom>
-}
+export default props => BaseButton(props)
```

Comment:
Should not use this since it does nothing. Also removes previous convenience:

`<Button ... />` styles could be shared between when they were a link or a form submit. Now, either have to set default classname in usage, or in two places.

Reply:
I kept the component because it might be useful to add event tracking for any button clicks.
Why do buttons need a default class, can they just make the default style for `button` instead of `.btn` ?
I'd rather if Link always meant link to another route and Button always for other actions. If the link looks like a button just give it a button class name. I find it easier to reason that "Button is always an html button, and Link is always an html a".

Reply:
buttons behave differently: https://stackoverflow.com/questions/3110564/remove-the-complete-styling-of-an-html-button-submit

Also, a are bound the enter key, button to space key.